### ISSUE TITLE
Dealing with variable-length sequence

### ISSUE THREAD

**Issue by zhh210** [1991viet/Hierarchical-attention-networks-pytorch#4]
Thanks for your work. I'm wondering if you plan to incorporate the variable length of each sentence/doc into the model training? It seems so far only padding is applied. To me it makes sense to pad the sequence but also true to let SGD only backprop through the actual length of the sequence. 

[https://discuss.pytorch.org/t/simple-working-example-how-to-use-packing-for-variable-length-sequence-inputs-for-rnn/2120]
